test/Microsoft.Build.Sql.Tests/DotnetTestBase.cs

```diff
@@ -88,6 +96,10 @@ protected void EnvironmentSetup()
             {
                 TestUtils.CopyDirectoryRecursive(this.CurrentTestDataDirectory, this.WorkingDirectory);
             }
+
+            // Add pkg folder as a nuget source
+            RunGenericDotnetCommand($"nuget add source \"{localNugetSource}\" --name TestSource_{TestContext.CurrentContext.Test.Name}", out _, out string stdError);
+            Assert.AreEqual("", stdError, "Failed to add local nuget source: " + stdError);
```

Comment:
get yourself a fluent assert library and level up from `AreEqual`

Reply:
Added #472